### ISSUE TITLE
Narrow scrollbars on Windows

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#5426]
https://forums.zotero.org/discussion/125769/small-slider-desktop-7-0-16-current-resolution-1920x1080-pixels

Not sure if we can control this, but from a quick check, it seems like our usable (not visible) width is 11px, Firefox's is 12px (?), and File Explorer is 17px.

**Comment by dstillman** [comment]
It looks like Edge's is also 12px, though, so this doesn't seem that non-standard. But maybe we can at least get that extra 1px.
